comp/core/tagger/tagger.go

```diff
@@ -136,7 +174,7 @@ func AccumulateTagsFor(entity string, cardinality collectors.TagCardinality, tb
 		}
 	}
 	mux.RUnlock()
-
+	// TODO(components) (tagger): defaultTagger is a legacy global variable to be eliminated
```

Comment:
Note: There is no need to have this comment each time `defaultTagger` is used having once is enough.